.github/linters/sun_checks.xml

```diff
@@ -32,167 +32,170 @@
+	<!-- <module name="Header"> -->
+	<!--   <property name="headerFile" value="${checkstyle.header.file}"/> -->
+	<!--   <property name="fileExtensions" value="java"/> -->
+	<!-- </module> -->
+
+	<module name="TreeWalker">
+
+		<!-- Checks for Javadoc comments.                     -->
+		<!-- See https://checkstyle.org/config_javadoc.html -->
+		<module name="InvalidJavadocPosition" />
+		<module name="JavadocMethod" />
+		<module name="JavadocType" />
+		<module name="JavadocVariable" />
+		<module name="JavadocStyle" />
+		<module name="MissingJavadocMethod" />
+
+		<!-- Checks for Naming Conventions.                  -->
+		<!-- See https://checkstyle.org/config_naming.html -->
+		<module name="ConstantName" />
+		<module name="LocalFinalVariableName" />
+		<module name="LocalVariableName" />
+		<module name="MemberName" />
+		<module name="MethodName" />
+		<module name="PackageName" />
+		<module name="ParameterName" />
+		<module name="StaticVariableName" />
+		<module name="TypeName" />
+
+		<!-- Checks for imports                              -->
+		<!-- See https://checkstyle.org/config_imports.html -->
+		<module name="AvoidStarImport" />
+		<module name="IllegalImport" /> <!-- defaults to sun.* packages -->
+		<module name="RedundantImport" />
+		<module name="UnusedImports">
+			<property name="processJavadoc" value="false" />
+		</module>
+
+		<!-- Checks for Size Violations.                    -->
+		<!-- See https://checkstyle.org/config_sizes.html -->
+		<module name="MethodLength" />
+		<module name="ParameterNumber">
+			<property name="max" value="10" />
+			<property name="tokens" value="METHOD_DEF" />
+		</module>
+
+		<!-- Checks for whitespace                               -->
+		<!-- See https://checkstyle.org/config_whitespace.html -->
+		<module name="EmptyForIteratorPad" />
+		<module name="GenericWhitespace" />
+		<module name="MethodParamPad" />
+		<module name="NoWhitespaceAfter" />
+		<module name="NoWhitespaceBefore" />
+		<module name="OperatorWrap" />
+		<module name="ParenPad" />
+		<module name="TypecastParenPad" />
+		<module name="WhitespaceAfter" />
+		<module name="WhitespaceAround" />
+
+		<!-- Modifier Checks                                    -->
+		<!-- See https://checkstyle.org/config_modifier.html -->
+		<module name="ModifierOrder" />
+		<module name="RedundantModifier" />
+
+		<!-- Checks for blocks. You know, those {}'s         -->
+		<!-- See https://checkstyle.org/config_blocks.html -->
+		<module name="AvoidNestedBlocks" />
+		<module name="EmptyBlock" />
+		<module name="LeftCurly" />
+		<module name="NeedBraces" />
+		<module name="RightCurly" />
+
+		<!-- Checks for common coding problems               -->
+		<!-- See https://checkstyle.org/config_coding.html -->
+		<module name="EmptyStatement" />
+		<module name="EqualsHashCode" />
+		<module name="HiddenField" />
+		<module name="IllegalInstantiation" />
+		<module name="InnerAssignment" />
+		<!-- <module name="MagicNumber" /> -->
```

Comment:
Avoid ignoring linters. Instead, fix the code quality, even if unrelated to the developed feature.